plugins/modules/sonic_lldp_interfaces.py

```diff
@@ -231,12 +290,20 @@
 #  no shutdown
 #  lldp transmit
 #  lldp tlv-set management-address ipv4 21.1.1.1
+#  lldp vlan-name-tlv allowed Vlan 10,15-20
+#  lldp vlan-name-tlv max-tlv-count 15
 # !
 # sonic#
 
   - name: Delete default LLDP Interface configurations
     dellemc.enterprise_sonic.sonic_lldp_interfaces:
       config:
+        - name: Ethernet0
+          vlan_name_tlv:
+            allowed_vlans:
+              - vlan: '10'
+              - vlan: '15-20'
```

Comment:
Here and for the other YAML sequences in the "Examples" section: Quotes are not needed for the values associated with the keys. The values are all implicitly handled as strings by the Ansible infrastructure.

```suggestion
            allowed_vlans:
              - vlan: 10
              - vlan: 15-20
```

Reply:
addressed